### PR TITLE
new readme

### PR DESCRIPTION
**Pull request by kslr**
Screenshot_2020-11-22 kslr kslr(4)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,68 @@
...
+
+### FHS-install-script
+_Maintainer: [@IceCodeNew](https://github.com/IceCodeNew)_
+
+```
+bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh)
+```
+
+### Docker
+_Maintainer wanted._
```
Comment (nicholascw) [inline README.md:31]: Maintainers

File: README.md [hunk 2]
```diff
@@ -1,21 +1,68 @@
...
+
+### Arch Linux
+_Maintainer: [@felixonmars](https://github.com/felixonmars)_
+
+```
+pacman -S v2ray
+```
+
+### Debian
+_Maintainer: [@rogers0](https://github.com/rogers0) [@ymshenyu](https://github.com/ymshenyu)_
```
Comment (nicholascw) [inline README.md:45]: Maintainers

File: README.md [hunk 3]
```diff
@@ -1,21 +1,68 @@
-# Project V
+<div>
+<img width="190" height="210" align="left"  src="https://raw.githubusercontent.com/v2fly/v2fly-github-io/master/docs/.vuepress/public/readme-logo.png" alt="V2ray"/>
```
Comment (nicholascw) [inline README.md:2]: alt="V2Ray"

File: README.md [hunk 4]
```diff
@@ -1,21 +1,68 @@
...
+</br>
+<h1>Project V</h1> 
+<p>Project V is a set of network tools that help you to build your own computer network.
```
Comment (nicholascw) [inline README.md:5]: thata help**s** you

File: README.md (hunk 1 above)
Comment (kslr) [reply README.md:31]: 都要改吗

File: README.md [hunk 5]
```diff
@@ -1,21 +1,68 @@
...
-[5]: https://godoc.org/v2ray.com/core?status.svg "GoDoc Badge"
-[6]: https://godoc.org/v2ray.com/core "GoDoc"
-[7]: https://goreportcard.com/badge/github.com/v2fly/v2ray-core "Goreportcard Badge"
-[8]: https://goreportcard.com/report/github.com/v2fly/v2ray-core "Goreportcard Result"
-[9]: https://img.shields.io/github/downloads/v2ray/v2ray-core/total.svg "v2ray/v2ray-core downloads count"
-[10]: https://github.com/v2ray/v2ray-core/releases "v2ray/v2ray-core release page"
-[11]: https://img.shields.io/github/downloads/v2fly/v2ray-core/total.svg "v2fly/v2ray-core downloads count"
-[12]: https://github.com/v2fly/v2ray-core/releases "v2fly/v2ray-core release page"
+## Related Links
+ - [Website](https://www.v2fly.org/) and [First Time Usage](https://www.v2fly.org/guide/start.html)
```
Comment (nicholascw) [inline README.md:15]: "Documentation" and "Newcomer's Instructions" to be more appropriate

File: README.md [hunk 6]
```diff
@@ -1,21 +1,68 @@
...
+ - Welcome to translate V2Ray via: **[Transifex](https://www.transifex.com/v2fly/public/)**
```
Comment (nicholascw) [inline README.md:16]: Translations for V2Ray documents are welcome. Visit {link} for more information.

btw link to the document repo where the more detailed readme is at could be better than directly forwarding people to Transifex.

File: README.md [hunk 7]
```diff
@@ -1,21 +1,68 @@
...
 
-Project V is a set of network tools that help you to build your own computer network. It secures your network connections and thus protects your privacy. See [our website](https://www.v2fly.org/) for more information.
+## Installation
+
+V2Ray is currently available and packaged for Debian/Ubuntu, Arch Linux, macOS, Windows. If you’re interested in packaging V2Ray for your distro, don’t hesitate to open an issue.
```
Comment (nicholascw) [inline README.md:20]: ~~V2Ray is currently available and packaged for Debian/Ubuntu, Arch Linux, macOS, Windows. If you’re interested in packaging V2Ray for your distro, don’t hesitate to open an issue.~~

V2Ray binaries are directly available in {releases/latest} as well as some major distros' repositories, including Debian, Ubuntu, Arch Linux, macOS (homebrew), etc. If you are willing to package V2Ray for other distros, you are also welcome to seek for help via our issues.

Comment (nicholascw) [reply README.md:20]: btw, most other open source projects' readmes would also provide a brief instruction of building from source.

File: README.md (hunk 5 above)
Comment (Loyalsoldier) [reply README.md]: I recommend this:

- Website: [https://www.v2fly.org](https://www.v2fly.org)
- Getting started with V2Ray: [https://www.v2fly.org/guide/start.html](https://www.v2fly.org/guide/start.html)
- Welcome to translate V2Ray documentations via **[Transifex](https://www.transifex.com/v2fly/public/)**

File: README.md [hunk 8]
```diff
@@ -1,21 +1,68 @@
...
+ - Welcome to translate V2Ray via: **[Transifex](https://www.transifex.com/v2fly/public/)**
 
-Project V is a set of network tools that help you to build your own computer network. It secures your network connections and thus protects your privacy. See [our website](https://www.v2fly.org/) for more information.
+## Installation
+
+V2Ray binaries are directly available in {releases/latest} as well as some major distros' repositories, including Debian, Arch Linux, macOS (homebrew), etc. If you are willing to package V2Ray for other distros, you are also welcome to seek for help via our issues.
+
+
+### FHS-install-script
+_Maintainers: [@IceCodeNew](https://github.com/IceCodeNew)_
```
Comment (Loyalsoldier) [inline README.md:24]: I suggest list the repo and its link first, then list maintainers like this:

```
### FHS-install-script

_Repo: [v2fly/fhs-install-v2ray](https://github.com/v2fly/fhs-install-v2ray)_
_Maintainers: [@IceCodeNew](https://github.com/IceCodeNew)_
```

### FHS-install-script

_Repo: [v2fly/fhs-install-v2ray](https://github.com/v2fly/fhs-install-v2ray)_
_Maintainers: [@IceCodeNew](https://github.com/IceCodeNew)_

**Comment by Loyalsoldier** [comment]
lint 我来吧

**Comment by Loyalsoldier** [comment]
新增了一点东西，这里看效果：https://github.com/Loyalsoldier/testREADME

**Comment by Loyalsoldier** [comment]
这个 lint 太奇怪了……

**Comment by kslr** [comment]
能不能简单添加上维护者的信息呢

**Comment by kslr** [comment]
图片

**Comment by kslr** [comment]
> 这个 lint 太奇怪了……

https://github.com/remarkjs/remark-lint/issues/178#issuecomment-476716698

**Comment by Loyalsoldier** [comment]
> > 这个 lint 太奇怪了……
> 
> [remarkjs/remark-lint#178 (comment)](https://github.com/remarkjs/remark-lint/issues/178#issuecomment-476716698)

这个写法太扯了……

**Comment by Loyalsoldier** [comment]
> 能不能简单添加上维护者的信息呢

- macOS:
  - Homebrew: `brew install v2ray` _[package maintainer: @kidonng ]_

这样？

**Comment by U-v-U** [comment]
我建议在 README 的 #Installation 节直接链接到 https://www.v2fly.org/guide/install.html 
在文档里写 Packagers / Maintainers

**Comment by kslr** [comment]
是的，如果要列的话可能要把所有人列出来

**Comment by kslr** [comment]
任何变化？

**Comment by Loyalsoldier** [comment]
在官网文档里添加 maintainer 吧（在项目首页似乎过于高调……

**Pull request merged**